ros2_control_demo_bringup/launch/rrbot_system_position_only_gazebo.launch.py

```diff
@@ -65,21 +67,16 @@ def generate_launch_description():
 
     # Initialize Arguments
     prefix = LaunchConfiguration("prefix")
-    use_fake_hardware = LaunchConfiguration("use_fake_hardware")
+    use_sim = LaunchConfiguration("use_sim")
     fake_sensor_commands = LaunchConfiguration("fake_sensor_commands")
     slowdown = LaunchConfiguration("slowdown")
     robot_controller = LaunchConfiguration("robot_controller")
 
     gazebo = IncludeLaunchDescription(
         PythonLaunchDescriptionSource(
             [
-                PathJoinSubstitution(
-                    [
-                        FindPackageShare("gazebo_ros"),
-                        "launch",
-                        "gazebo.launch.py",
-                    ]
-                )
+                os.path.join(get_package_share_directory("gazebo_ros"), "launch"),
```

Comment:
Why change this? I would not use os.path.jont, but rather `FindPackageShare`. This is basically doing the same as your changes.

Reply:
Fixed by 0383c6d